sartools/abims_sartools_deseq2.xml

```diff
@@ -45,26 +45,26 @@
             </param>
             <when value="hide" />
             <when value="show">
-		<expand macro="batch_param" />
```

Comment:
Can you use a section instead of `<conditional name="advanced_parameters" >` ?

Reply:
done